main.go

```diff
@@ -66,6 +67,19 @@ func main() {
 	s.Engine.POST("/invites/:invite_id/deny", s.invitesHandler.DenyInvite)
 	s.Engine.GET("/invites", s.invitesHandler.ListInvites)
 
+	// Notes
+	s.Engine.GET("/notes/:note_id", s.notesHandler.GetNote)
+	s.Engine.POST("/notes", s.notesHandler.CreateNote)
+	s.Engine.PATCH("/notes/:note_id", s.notesHandler.UpdateNote)
+	s.Engine.DELETE("/notes/:note_id", s.notesHandler.DeleteNote)
+	s.Engine.GET("/notes/:author_id", s.notesHandler.ListNotes)
```

Comment:
That's not how I think we should handle the listing of notes. Prefer using query parameters in case we want to add more filters in the future.

```
GET /notes?author_id=...&other_filter=...
```